backend/curriculum_tracking/tests/test_activity_log_entry_creators_envoked.py

```diff
@@ -106,3 +106,94 @@ def test_add_review(self):
             start_url, data={"status": COMPETENT, "comments": "blah more stuff"}
         )
         self.assertEqual(LogEntry.objects.count(), 2)
+
+
+class log_card_review_requested_Tests(APITestCase, APITestCaseMixin):
+    LIST_URL_NAME = "agilecard-list"
+    SUPPRESS_TEST_POST_TO_CREATE = True
+    SUPPRESS_TEST_GET_LIST = True
+
+    def test_review_requested_authorised_user(self):
+        actor_user = UserFactory(is_superuser=True, is_staff=True)
+        content_item = factories.ProjectContentItemFactory(
+            project_submission_type=ContentItem.LINK, template_repo=None
+        )
+        card = factories.AgileCardFactory(
+            status=AgileCard.READY,
+            recruit_project=factories.RecruitProjectFactory(content_item=content_item),
+            content_item=content_item,
+        )
+        self.login(actor_user)
+        card.start_project()
+        self.assertEqual(card.status, AgileCard.IN_PROGRESS)
```

Comment:
@Samantha-Hampton The only reason the login is need is because of the post you're making later in line 131, the assert for the `card.status` isn't affected by this and will pass even without the login.